Services/Authentication/classes/class.ilAuthWeb.php

```diff
@@ -28,7 +28,7 @@
  * 
  *  @ingroup ServicesAuthentication
  */
-class ilAuthWeb extends Auth
+class ilAuthWeb
```

Comment:
Same here. while reading this Class the ArtifactBuilder just dies since Auth is not available. Is this class even used?